lib/filesystem_utils.pm

```diff
@@ -430,7 +463,18 @@ sub validate_lsblk {
     }
 
     for my $col (keys %{$validation_test_data}) {
-        if ($validation_test_data->{$col}{value} ne $blockdev->{$col}) {
+        if ($col eq 'mountpoints') {
+            my ($primary_mountpoint) = grep { /^\/$/ } @{$blockdev->{$col}};
```

Comment:
There is no such thing as a "primary mountpoint" and FWICT this is just a hack to treat `/` specially.

IMO this should check whether `$blockdev->{$col}` contains `$validation_test_data->{$col}{value}`, or if the latter is an array, all of them.

Reply:
right, there is more straightforward to extract. The array case I will keep it out, we don't have such cases afik and the data structure doesn't allow to use it, but it is a good point to remember, in the future we might change this validation and even use findmount as mountpoints is not very useful, if instead of showing us the first mounted is showing the last which could be any of the subvolume, is not primary but the one that is specified during installation.

Reply:
Yup, sounds good.